src/powfacpy/plot_interface.py

```diff
@@ -220,6 +220,10 @@ def _set_axis_attributes(self, axis, kwargs):
     for attribute, value in kwargs.items():
       axis.SetAttribute(attribute, value)      
 
+  def set_x_axis_range_of_active_plot(self,range: Iterable):
```

Comment:
this is redundant with 
`set_x_axis_attributes(rangeMin=min, rangeMax=max)`,
that should be used instead

Reply:
I think it is ok to keep both and the user can decide which one to use. To use set_x_axis_attributes you need to look up the keywords rangeMin and rangeMax, so this method is a bit easier to use in my view.